readthedocs/projects/models.py

```diff
@@ -215,8 +212,11 @@ class AddonsConfig(TimeStampedModel):
     search_enabled = models.BooleanField(default=True)
     search_default_filter = models.CharField(null=True, blank=True, max_length=128)
 
-    # Stable/Latest version warning
-    stable_latest_version_warning_enabled = models.BooleanField(default=True)
+    # Notifications
+    notifications_enabled = models.BooleanField(default=True)
+    notifications_show_on_latest = models.BooleanField(default=True)
+    notifications_show_on_non_stable = models.BooleanField(default=True)
+    notifications_show_on_external = models.BooleanField(default=True)
```

Comment:
All these should be nullable to avoid downtime.

https://dev.readthedocs.io/en/stable/migrations.html#adding-a-new-field

If we are okay with not allowing to create/edit this model while the migration takes place, that's okay, I guess.

Reply:
Seems fine, it should be a quick migration?

Reply:
I do wonder if we want to default all of these to `True`, though? In particular, I'm not sure we want `latest` to have a default warning?

Reply:
If the field has a default value it doesn't need to be nullable, right? Once the field is added, it will be the default value automatically and when creating a new row from an old instance without passing the new value.

Reply:
I'm happy to talk more about this and change this default now and/or in the future, tho.

Reply:
Good point 👍🏼 . We should mention that in that document to avoid this confusion every time 😄 .